api_tests/preprint_providers/views/test_preprint_provider_detail.py

```diff
@@ -63,6 +72,38 @@ def test_preprint_provider_exists(self, app, provider_url, fake_url):
             expect_errors=True)
         assert taxonomies_res.status_code == 404
 
+    def test_preprint_provider_exists_for_generalized_endpoint(self, app, provider_url_generalized, fake_url_generalized):
```

Comment:
These look like a direct copy of the provider_exists tests above, if so, it might be better to parametrize the url and have a single test method; https://docs.pytest.org/en/latest/parametrize.html

something like this:
```python
@pytest.mark.parametrize("test_url,fake_url", [
    (provider_url_generalized, fake_url_generalized), 
    (provider_url, fake_url)
]
def test_preprint_provider_exists(self, test_url, fake_url, app):
```